### ISSUE TITLE
build-s3-dist.sh and README is wrong if you want to deploy from source

### ISSUE THREAD

**Issue by MoJo2600** [awslabs/aws-connected-vehicle-solution#4]
The steps to deploy the solution from source are not correct.

* The build-s3-dist.sh script will overwrite the name of your custom bucket and with that, the lambda sources are always loaded from the aws bucket called `solutions`
* The name of the CloudFormation template is wrong in the build-s3-dist.sh script
* The documentation on how to deploy your own stack is wrong in the README. The bucket name has to be suffixed with the region, as the CloudFormation templates always adds the region to the bucket prefix

**Comment by beomseoklee** [comment]
We apologize for the inconvenience, and we thank for your contribution. We will research this one.

**Comment by mariano-calandra-xp** [comment]
I had it working in this way. I created a bucket named `<my_bucket_name>-<current_region>` (for instance `project-eu-west-1`. Then I exported the bucket prefix (i.e. the bucket name without `-<current_region>`):

```
export BUCKET_PREFIX=<my_bucket_name>
```

As last step, in `build-s3-dist.sh` I changed:

```
cp ./connected-vehicle-platform.yaml ./dist/aws-connected-vehicle-solution.template
export BUCKET_PREFIX=solutions-test
if [ $PIPELINE_TYPE = "release" ]; then
    export BUCKET_PREFIX=solutions
fi
replace="s/%%BUCKET_NAME%%/$BUCKET_PREFIX/g"
sed -i '' -e $replace dist/aws-connected-vehicle-solution.template
```

in 

```
cp ./aws-connected-vehicle-solution.template ./dist/aws-connected-vehicle-solution.template
sed -i "s/%%BUCKET_NAME%%/$BUCKET_PREFIX/g" ./dist/aws-connected-vehicle-solution.template
```

In the README.md is mispelled the copy part towards S3:

```
aws s3 cp ./dist s3://<my_bucket_name>-<current_region>/connected-vehicle-solution/latest --recursive --profile <the_profile_you_need>
```

Deploying these artifacts no error is reported on CloudFormation but the stack is stuck (pun intended) on CREATE IN PROGRESS. Any idea?

**Comment by mobri2a** [comment]
Fixed in latest release (v2.1.2)
